src/config_template.json

```diff
@@ -7,5 +7,12 @@
     "random": true,
     "targetAmountOfHumanResources": 10,
     "targetAmountOfExhaustibleResources": 10
+  },
```

Comment:
I'd like to have a standard config file for unix based systems inside the repossitory.